src/ui/assets-and-images.md

```diff
@@ -162,14 +112,26 @@ Future<String> loadAsset() async {
 
 ### Loading images
 
-Flutter can load resolution-appropriate images for
-the current device pixel ratio.
+To load an image, use the [`AssetImage`][]
+class in a widget's `build()` method.
 
-#### Declaring resolution-aware image assets {#resolution-aware}
+For example, your app can load the background
+image from the asset declarations above:
```

Comment:
**issue:** Don't use above per [GSG](https://developers.google.com/style/word-list#above).
```suggestion
For example, your app can load the background
image from the previous asset declarations:
```

Reply:
Addressed with commits [fix image asset loading example](https://github.com/flutter/website/pull/8710/commits/92306970835201e0ce8fd2e33c746a1b40616438) and [update BackgroundImage docregion](https://github.com/flutter/website/pull/8710/commits/fae0b3c8a3eca2feb98e60928d7875e30704e639)